index.html

```diff
@@ -66,24 +66,32 @@ <h1 class="front-page__title">Ask me question</h1>
     <form class="question-page__question-container" method="dialog">
       <div class="question-page__question-form">
 
-        <input type="text" id="questionInput" class="question-input"  placeholder="Ask me question">
-        <input class="question-page__clearButton" type="reset" value="Clear" />
-        
+        <input type="text" id="questionInput" class="question-input" placeholder="Ask me question">
+        <button class="question-page__clearButton material-symbols-outlined" type="reset">
+          disabled_by_default
+        </button>
+
       </div>
     </form>
 
     <div class="tellme-container">
 
-      <button class="tellme-firstButton">Tell me fun fact</button>
-      <button class="tellme-secondButton">Tell me a joke</button>
+      <button id="factButton" class="tellme-firstButton">Tell me fun fact</button>
+      <button id="jokeButton" class="tellme-secondButton">Tell me a joke</button>
 
     </div>
 
-    <div class="question-page__answer"></div>
+    <div class="question-page__answer">
+
+      <div id="answerOutput" class="answerContainer">
+
+      </div>
+
+    </div>
 
   </div>
 
```

Comment:
Keep the enters in a convention. You have habit to leave unnecesary white spaces. they make files larger.